### DISCUSSION TITLE
types for is_js incorrectly named [@types/is]

### DISCUSSION THREAD

**Discussion by valepu** [DefinitelyTyped/DefinitelyTyped#56365] [Issues with a @types package]
Hi,
I want to report an error in the name of the types for [is_js](https://github.com/arasatasaygin/is.js) as defined here https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/is . They are called @types/is and this causes some issues when importing it. 

>  if i import it like this:
> 
> import * as is from 'is_js';
> 
> i get an error saying it can't find the type definition when building (despite VS code not giving me any error). if i import it like this
> 
> import * as is from 'is';
> 
> I get a different error. The solution seems to be to install is_js with an alias: "is": "npm:is_js@^0.9.0", and then import the module like this import * as is from 'is';

from my issue at https://github.com/arasatasaygin/is.js/issues/322 
You can also see from the issue that "is" is a different library than "is_js" so this might cause further confusion.
I also want to add that even though the library is called is.js on github, when installing it it's called is_js

[a comment by an automated account is not shown]

**Comment by gricey432** [comment]
I'm not really sure why it is named this way, maybe Rodrigo will have a better idea.

I stopped using is_js a few years ago, but the types were working back then and haven't been changed in 3 years. These types have 10s of thousands of weekly downloads so I imagine there would have been bug reports in the last few years if they didn't work.

Possibly something has changed in newer TS versions? Or maybe something with your setup?
